packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -648,6 +648,12 @@ export class DeltaManager
                     this.client.mode = requestedMode;
                     connection = await docService.connectToDeltaStream(this.client);
                 } catch (origError) {
+                    if (origError?.errorType === DriverErrorType.sessionForbidden) {
```

Comment:
Please add to IF checks:
typeof origError === "objet" && origError !== null
Exception can be anything in JS, including number :)